virt/virt-2-4-release-notes.adoc

```diff
@@ -155,37 +200,19 @@ value by running `oc describe node <node>` for all nodes and looking at the
 `cpu-model-<name>` labels. Select the CPU model that is present on all of your
 nodes.
 
-* When running `virtctl image-upload` to upload large VM disk images in `qcow2`
-format, an end-of-file (EOF) error may be reported after the data is
-transmitted, even though the upload is either progressing normally or completed.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1789093[*BZ#1789093*])
-+
-Run the following command to check the status of an upload on a given PVC:
-+
-----
-$ oc describe pvc <pvc-name> | grep cdi.kubevirt.io/storage.pod.phase
-----
-
```

Comment:
Just making sure that this was removed on purpose (note above says this section is unchanged)

Reply:
This content and some of the other missing content pieces that you mentioned in this PR - not sure why they are not present in the PR content. Added this particular content in the location you noted above in your comments.